presto-docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -259,6 +259,77 @@ Field           Required  Type      Description
 
 There is no limit on field descriptions for either key or message.
 
+Row Encoding
+------------
+
+For key and message, an encoder is used to map message and key data onto table columns.
+
+The Kafka connector contains the following encoders:
+
+* ``csv`` - Kafka message is formatted as comma separated message, and fields are mapped to table columns
+* ``avro`` - Kafka message is mapped to Avro fields based on an Avro schema
+
+.. note::
+
+    A table definition file must be defined for the encoder to work.
```

Comment:
Link to the section and also detail there how that is supposed to be done? E.g. where do you put the definition file once you written it and how to you configure Presto to be aware of it and use it.